backend/src/main/java/hu/flowacademy/meteo/controller/SoilDataController.java

```diff
@@ -0,0 +1,28 @@
+package hu.flowacademy.meteo.controller;
+
+import hu.flowacademy.meteo.dto.BetweenDatesDto;
+import hu.flowacademy.meteo.dto.MeasurmentDto;
+import hu.flowacademy.meteo.dto.SoilDataDto;
+import hu.flowacademy.meteo.service.MeasurmentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+public class SoilDataController {
```

Comment:
Amikor a 3. teljesen hasonló controllert látja az ember, akkor eszébe jut, hogy ezek valaha bővülni fognak és nagyok lesznek, vagy elsült a copy-pase? Ha ezekről nem látod, hogy jelentősen bővülnének a jövőben, akkor vond össze őket egy DataController-be, és kerüljön bele az összes.